tests/test_season.py

```diff
@@ -0,0 +1,82 @@
+from datetime import datetime
+
+import pandas as pd
+
+from hdc.algo.season import Season
+
+# pylint: disable=use-implicit-booleaness-not-comparison
+
+
+def test_season():
+    # Test initialization and basic attributes
+    season = Season([(1, 10), (11, 20), (21, 30)])
```

Comment:
Let's discuss the implementation approach. Here a `Season` instance is an object that can map dates to specific seasons in time. Compare this to the `Dekad` class, where each instance is actually a specific Dekad in time. I think we want to replicate the latter approach.